pencil2d.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.1
+Type=Application
+Name=Pencil2D
+GenericName=Animation Software
+Comment=Create traditional hand-drawn animation using both bitmap and vector graphics
+Icon=pencil2d
+Exec=Pencil2D
```

Comment:
Actually the point of that sentence is not to avoid some dangerous name clashes or anything, in fact the reason for discouraging redundancy is pretty boring: Because it doesn’t provide any value. I believe you are mostly a macOS guy, so I suppose you are about as clueless about the Linux ecosystem as I am about the macOS ecosystem. Desktop Entries on Linux are somewhat like shortcuts on Windows or (from what I read on Wikipedia) Aliases on macOS. They contain information about a certain target such as an application or a link, and they are often placed in a central location which serves as a database of sorts. The Desktop Entries in that location can then be read by programs to find suitable applications for opening certain file types or to provide application launchers, among other things. That’s where Keywords comes in. Sometimes you might look for an application but can’t quite remember which one it was. In that case you can often search through available applications, mainly through Name (e.g. Firefox) and GenericName (e.g. Web Browser). Keywords however is for cases where you want to provide additional searchable key words, so that the entry can be found with not-so-specific searches.
So putting animation in Keywords isn’t very helpful simply because it can already be found in GenericName.

For reference, this is how the specification explains Keywords:

> A list of strings which may be used in addition to other metadata to describe this entry. This can be useful e.g. to facilitate searching through entries.

Whew, that turned into a novel!